ui/app/AppLayouts/Communities/panels/OverviewSettingsPanel.qml

```diff
@@ -329,6 +339,7 @@ StackLayout {
 
             active: !!editSettingsPanelLoader.item &&
                     editSettingsPanelLoader.item.dirty
+            visible: active
```

Comment:
Here the `SettingsDirtyToastMessage` was obscuring the bottom part where the Sharding section resides because it only sets the `opacity` but doesn't set `visible: false`